test/integration/test/captions.test.js

```diff
@@ -1,12 +1,24 @@
 const clientHelper = require('../src/client-helper');
 const configHelper = require('../src/config-helper');
+const InstanceStateError = require('../../../src/models/RevAiApiError').InvalidStateError;
 
 test('Can get srt captions', async (done) => {
     const client = clientHelper.getClient(configHelper.getApiKey());
     const jobList = await client.getListOfJobs();
     const jobId = clientHelper.getTranscribedJobId(jobList);
     expect(jobId).toBeDefined();
-    const captionsStream = await client.getCaptions(jobId);
+    var captionStream = null;
```

Comment:
I don't get this change at all, what does it have to do with the issue?

Reply:
Nothing, it's irrelevant to the issue but was causing certain integration tests which should'nt be failing to fail. The integration test picks a job at random (that is transcribed and async) and tries to get captions output. In some cases this job was a speaker channel job so the call would fail

Reply:
It's a small fix so I just made it. Do you think it needs it's own issue?

Reply:
Not really, just make sure to make a comment about off-pr changes so people know what's going on